pages/index.tsx

```diff
@@ -24,57 +27,149 @@ export default function Home() {
       />
 
       <main>
-
         <HeroBanner {...content.heroBanner} aria-label={content.heroBanner.ariaLabels?.ctaButton} />
 
-        <SectionContainer id={content.ourMission.id}>
-          <PaperSection>
-            <Grid container id={`${content.ourMission.id}-main-grid`}>
-              <Grid item xs={12}>
-                <Typography variant="h2" sx={{ textAlign: 'center' }}>
-                  {content.ourMission.header}
-                </Typography>
-                <MuiMarkdown>{content.ourMission.body}</MuiMarkdown>
-                <br />
-                <br />
-                <MuiMarkdown>{content.ourMission.body2}</MuiMarkdown>
-              </Grid>
+        <SectionContainer>
```

Comment:
Adding a maxWidth of "md" to this section, and probably the other section containers should help adjust the width of everything and match it closer to the figma. Right now everything is too wide.

Reply:
@aitzeng could you do this one please

Reply:
Section container maxWidth set to md: 936px